src/ai/backend/storage/types.py

```diff
@@ -62,11 +62,13 @@ def as_trafaret(cls) -> t.Trafaret:
 @attr.s(auto_attribs=True, slots=True, frozen=True)
 class VFolderCreationOptions:
     quota: int
+    bsize: str
 
     @classmethod
     def as_trafaret(cls) -> t.Trafaret:
         return t.Dict({
             t.Key('quota', default=0): t.ToInt[0:],
+            t.Key('bsize', default=''): t.String,
```

Comment:
You could use `ai.backend.common.validators.BinarySize` for explicit conversion of human-readable byte-size string representations into integers.